### ISSUE TITLE
Feature Request: Purchase/Consume by Name

### ISSUE THREAD

**Issue by kses** [PantryParty/pantry_party#90]
I would like to be able to consume (all 3 variants) and purchase items by name for items without barcode and without switching to stock view. 

I have a suggestion of how it could be implemented in the ui.

I imagine it could look something like this (but nicer).

<img width="400" alt="image" src="https://user-images.githubusercontent.com/17168512/84604131-26f1e000-ae94-11ea-9c55-fbc18e6b97f0.png">

Takes you to window similar to the current stock window but with new name, for purchase it removes all filters and show all items including 0 stock items, for consume it shows only product in stock. And a add button would have to be added to add new products without barcode.

Taping a product would be the same as scanning.

<img width="412" alt="image" src="https://user-images.githubusercontent.com/17168512/84604285-302f7c80-ae95-11ea-84db-cc659b1e2891.png">

**Comment by arjes** [comment]
Good idea thanks. For now you can perform these actions from the current
stock page

**Comment by kses** [comment]
The consume I get in the current stock and that is great but not purchase right?

![IMG_4281](https://user-images.githubusercontent.com/17168512/84604664-e005e980-ae97-11ea-8d39-0e246a1d7687.png)

**Comment by arjes** [comment]
Bah you are right sorry! And your buttons look weird too. What device are
you using?

**Comment by kses** [comment]
iphone xs, ios 13.5.1
I can not find anywhere to check the pantryparty version number it is set to autoupdate.
